packages/analytics-js/src/services/HttpClient/xhr/xhrRequestHandler.ts

```diff
@@ -79,6 +76,9 @@ const xhrRequest = (
     };
```

Comment:
you mean rejet with xhr object as returned value instead of throwing an error like?

```
new Error(`Request failed with status: ${xhr.status}, ${xhr.statusText} for URL: ${options.url}`)
```

Reply:
Yes, with at least the actual status code.

Reply:
on it

Reply:
so you want to remove the error handling from HttpClient alltogether?